bot_components/slack_comm.py

```diff
@@ -8,7 +8,7 @@
 """
 import json
 
-import bot_components
+from .. import bot_components
```

Comment:
@bboe : I think I am on the right path now. Making some good progress progress.

Need to have a `plugins/hubcommander` directory, and need to have the imports everywhere be along the lines of: `plugins.hubcommander.path.to.whatever` <-- this is how i prefer the imports to be.